src/main.ts

```diff
@@ -1,3 +1,4 @@
+import 'hammerjs';
```

Comment:
where do you use this?

Reply:
I don't use it now, but I added it because it was one of the steps in documentations Angular material. I thought, that we can use it in the future tasks.
Should I remove it?

Reply:
hammerjs is used for gestures, we are not going use them

Reply:
removed hammerjs